build.gradle

```diff
@@ -53,6 +57,46 @@ buildscript {
         return audioSwitchBintrayPassword;
     }
 
+    ext.getAudioSwitchJfrogOssUsername = {
+        def audioSwitchJfrogOssUsername  = System.getenv("AUDIOSWITCH_JFROG_OSS_USERNAME");
```

Comment:
Do these credentials exist in the shared OnePassword vault? It would be handy to have them there for the rest of the team.

Reply:
Yes it does. See the Twilio SDK Build Bintray user

Reply:
Great! Thanks.